runelite-client/src/test/java/net/runelite/client/plugins/specialcounter/SpecialCounterPluginTest.java

```diff
@@ -258,6 +258,11 @@ public void testNotification()
 		specialCounterPlugin.onVarbitChanged(new VarbitChanged());
 		specialCounterPlugin.onHitsplatApplied(hitsplat(target, Hitsplat.HitsplatType.DAMAGE_ME));
 
+		// Set up spec weapon as BGS(OR)
+		ItemContainer equipment = mock(ItemContainer.class);
+		when(equipment.getItem(EquipmentInventorySlot.WEAPON.getSlotIdx())).thenReturn(new Item(ItemID.BANDOS_GODSWORD_OR, 1));
+		when(client.getItemContainer(InventoryID.EQUIPMENT)).thenReturn(equipment);
+
```

Comment:
Instead of adding this to an existing test, another test should be added to verify specs from varied weapon types are (a) identified as special attacks, and (b) stacked instead of tracked separately.